user_modules/shared_testcases/testCasesForPolicyAppIdManagament.lua

```diff
@@ -0,0 +1,77 @@
+---------------------------------------------------------------------------------------------
+-- Policy: AppID Management common module
+---------------------------------------------------------------------------------------------
+
+local common = {}
+
+local commonFunctions = require("user_modules/shared_testcases/commonFunctions")
+
+local policy_file_path = commonFunctions:read_parameter_from_ini("SystemFilesPath")
```

Comment:
@istoimenova  Please comment call of method read_parameter_from_ini because now this method in separate branch and now this changes breake implementation in this branch. Please restore old string and comment call of method.
For example: local policy_file_path = hard_code_path --commonFunctions:read_parameter_from_ini("SystemFilesPath")
and this script must wait for merge common function. After merge need to uncomment, and use only common function